app/src/main/java/io/github/droidkaigi/confsched2018/presentation/map/MapFragment.kt

```diff
@@ -128,6 +128,6 @@ class MapFragment : Fragment(), Injectable, OnMapReadyCallback {
         fun newInstance(): MapFragment = MapFragment()
 
         private const val placeLat = 35.6957954
-        private const val placeLang = 139.69038920000003
+        private const val placeLng = 139.69038920000003
```

Comment:
Also, if you do not mind, can you capitalize it?
https://android.github.io/kotlin-guides/style.html#constant-names

Reply:
Oh, I understood. I'll give it a try!